src/metadata/DefaultMetadata.js

```diff
@@ -0,0 +1,23 @@
+/**
+* Class that represent the metadata of a specific implementation.
+*/
+class DefaultMetadata {
+  /**
+  * Default constructor.
+  */
+  constructor() {
```

Comment:
`constructor(metadata)` where `metadata` is the value of the metadata provide by leto-modelizer